README.md

```diff
@@ -16,10 +15,24 @@ R_CONFIG_ACTIVE=YYYYQQ
   - ar_company_id__factset_entity_id e.g. "2022-08-17_rmi_ar_fs_id_bridge_2021q4.csv"
 - `HOST_OUTPUTS_PATH` the local path to where the output files should be saved. This directory will be mounted into the Docker image and it will write some files to this directory, so appropriate permissions in this directory will be required.
 - `GITHUB_PAT` valid GitHub PAT that grants access to the repos [RMI-PACTA/pacta.scenario.preparation](https://github.com/RMI-PACTA/pacta.scenario.preparation) and [RMI-PACTA/pacta.data.preparation](https://github.com/RMI-PACTA/pacta.data.preparation)
-- `R_DATABASE_USER` username for the FactSet database
-- `R_DATABASE_PASSWORD` password for the FactSet database
 - `R_CONFIG_ACTIVE` the name of the config to be used. The [config.yml](config.yml) file contains multiple named configurations which can be named here to activate them. Current options are: `2021Q4`, `2021Q4_dev_vm`, `2021Q4_dev_local`, `2021Q4_prod_vm`, `2022Q2`
 
 Run `docker-compose up` from the root directory, and docker will build the image (if necessary), and then run the data.prep process given the specified options in the .env file.
 
 Use `docker-compose build --no-cache` to force a rebuild of the Docker image.
+
+## Required Input Files
+
+### FactSet Data
+
+Files exported by [`{workflow.factset}`](https://github.com/RMI-PACTA/workflow.factset) provide financial data to tie to production data.
+See the [`workflow.factset` README](https://github.com/RMI-PACTA/workflow.factset#exported-files) for more information on expected file format.
```

Comment:
Some input files are already defined explicitly in the `README.md` above (under `HOST_INPUTS_PATH`).
I would say combine these with the list above or combine the list above down here, but it's confusing to mention input datasets in two different places.

My preference would be the latter.